src/App/docutils/ComponentPreview/components/ShowCasePanelAdvanced.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import { Topbar } from "./Topbar";
+import { ComponentDescription } from "./ComponentDescription";
+import { Sidebar } from "./Sidebar";
+import { Preview } from "./Preview";
+
+export const ShowCasePanelAdvanced = ({
+    showCasePanelAdvanced,
+    children,
+    activeTab,
+    updateActiveTab,
+    activeOptions,
+    updateActiveOptions,
+    staticPreview,
+    previewMinHeight
+}) => {
+    const [sidebarDisplayed, setSidebarDisplayed] = useState(false);
+
+    // TODO: could this be done via CSS instead ?
```

Comment:
Could it? 🤔

Reply:
I'm not sure. I wish.
but yes I do think it should be possible,
either via good-ol media-query, or now with the shiny container-queries
anyway let's remove this comment, and fix this later if it needs to be

🧼🧽 boop 🧹cleaned 👍